packages/atlas/src/theme/web/exclusion-variables.scss

```diff
@@ -1,3 +1,6 @@
+//== Optimize atlas for modern client (not compatible with dojo client)
+$use-modern-client: false;
```

Comment:
How can I learn what changes/sets this value? What sets it to true?

Reply:
https://mendix.slack.com/archives/C8AF2EVEW/p1693815542792339
In short. This will be changed by the end user (developer).

Reply:
don't forget the versioning for atlas-core.
due to the introduction of this variable, I think we can put minor increase to 3.12